hazelcast/proxy/cp/atomic_reference.py

```diff
@@ -54,7 +54,9 @@ class AtomicReference(BaseCPProxy["BlockingAtomicReference"], typing.Generic[Ele
     server-side setting.
     """
 
-    def compare_and_set(self, expect: ElementType, update: ElementType) -> Future[bool]:
+    def compare_and_set(
+        self, expect: typing.Optional[ElementType], update: typing.Optional[ElementType]
```

Comment:
Why did we change these? These are also public API

Reply:
The type hints are introduced in this release, so not a breaking change. These were all optional (as it is now) but I missed that initially